packages/server/src/plugin/landingPage/default/types.ts

```diff
@@ -78,8 +90,45 @@ export interface ApolloServerPluginEmbeddedLandingPageProductionDefaultOptions
    * Users can configure their landing page to render an embedded Explorer.
    */
   embed: true | EmbeddableExplorerOptions;
+  /**
+   * The ID of a collection, paired with an operation ID to populate in the Sandbox on load.
+   *
+   * You can find these values from a registered graph in Studio by
+   * clicking the ... menu next to an operation in the Explorer of that graph and
+   * selecting View operation details.
+   */
+  defaultOperation?: {
+    collectionId: string;
+    operationId: string;
+  };
 }
 
+type EmbeddableSandboxOptions = {
+  initialState?: {
+    /**
+     * If true, the embedded Sandbox periodically polls your initialEndpoint for schema updates.
+     *
+     * The default value is true.
+     */
+    pollForSchemaUpdates?: boolean;
+    /**
+     * Headers that are applied by default to every operation executed by the landing page's Sandbox.
+     * Users can disable the application of these headers, but they can't modify their values.
+     *
+     * The landing page's Sandbox always includes these headers in its introspection queries
+     * to your endpoint.
+     */
+    sharedHeaders?: Record<string, string>;
+  };
```

Comment:
@cheapsteak @trevor-scheer noting a weirdness here. 

This was added in response to [this issue](https://github.com/apollographql/embeddable-explorer/issues/214), where users expect `headers` to be applied to their introspection request. This would be a major update b/c the behavior would change with no config change. I added `sharedHeaders` as an option instead, but it will be confusing for folks to have a top level `headers` option and then  a `sharedHeaders` option nested in `embed: { initialState: { sharedHeaders: {} } }`. I don't see a way around this but will add a `TODO` for the next major version perhaps? What do you think?

Reply:
What's the desired outcome for the API? A `TODO(as5), `// @deprecated`, and an actual `console.warn` deprecation warning when we detect bad usage with helpful messaging would be good if people can migrate to a correct thing now.

If they can't migrate to the thing we want it to be yet, a `TODO(as5)` w/comment is sufficient.